src/sections/Article/index.module.scss

```diff
@@ -4,6 +4,10 @@
   padding: var(--space-48) var(--space-32) var(--space-48) var(--space-32);
   width: calc(100vw - 36rem);
 
+    background: url("/static/images/header.svg");
+    background-repeat: no-repeat;
+    background-size:contain;
```

Comment:
Looks like Prettier is failing in CI, Do `npm run format` to fix it.

Reply:
I.D.K how to fix

```bash
src/sections/Article/index.module.scss
 7:3  ✖  Expected background to come before width  order/properties-alphabetical-order

1 problem (1 error, 0 warnings)
```

Reply:
Try swapping line no.8 and 9

Reply:
Looks like you need to move line no. 7,8,9 before `width` property

Reply:
Oh ok